gui/pages.py

```diff
@@ -23,14 +24,24 @@ class Menu(Frame):
     def __init__(self, parent_container, controller):
         super().__init__(parent_container)
 
-        InputFile(self).render_input_file_widgets(1)
```

Comment:
replace anonymous calls with variables. Probably irrelevant but we'll see how we can make use of variables. If not then just reverse this update to save more lines.